pkg/streamingpromql/engine_test.go

```diff
@@ -2381,6 +2384,17 @@ func TestAnnotations(t *testing.T) {
 				`PromQL warning: quantile value should be between 0 and 1, got 1.5 (1:10)`,
 			},
 		},
+		"double_exponential_smoothing() with float and native histogram at same step": {
+			data:                    `some_metric 10 {{schema:0 sum:1 count:1 buckets:[1]}}`,
+			expr:                    "double_exponential_smoothing(some_metric[1m1s], 0.5, 0.5)",
+			expectedInfoAnnotations: []string{`PromQL info: ignored histograms in a range containing both floats and histograms for metric name "some_metric" (1:30)`},
+		},
+		"double_exponential_smoothing() with only native histogram at same step will result with no annotations": {
```

Comment:
[nit] We haven't included the expected behaviour in the test names elsewhere in this method:

```suggestion
		"double_exponential_smoothing() with only native histogram at same step": {
```

Reply:
Addressed in 4e76aca6303b7d2fb9727105298032bbea8d57c6